src/iperf_client_api.c

```diff
@@ -582,4 +582,12 @@ iperf_run_client(struct iperf_test * test)
     iflush(test);
 
     return 0;
+
+  cleanup_and_fail:
+    iperf_printf(test, "cleanup_and_fail\n");
```

Comment:
Hi, this added output also when using the JSON output format, breaking parsing of the resulting JSON output (not a big issue in practice as there was some error anyways). Could this printf just be dropped?

Reply:
Wow that's a pretty rookie mistake...I shouldn't have left that debugging code in there (and to be clear this is totally my fault...it wasn't a part of the original patches in #1023).  I'll get rid of it.  Good catch, thanks for the review.